src/train.py

```diff
@@ -45,10 +45,14 @@
 # Create sample and checkpoint directories
 os.makedirs("saved_models/%s" % opt.model_name, exist_ok=True)
 
+# finds all possible transfer pairings without repetition (each will be used for a forward and backward pass)
+transfer_combos = list(itertools.combinations(range(opt.n_spkrs), 2))
```

Comment:
This will be very intense number of computations. I don't think we need to go over all the possible pairs. making it heuristic will be somewhat equivalent and will save a lot of computations. that was I would think about is that for `id_1` you make sure you go over all of the speakers, and for `id_2` you randomly select another speaker. Might take longer to converge but will be more efficient in my opinion.

Reply:
Updated! Just noting here that I decided to iterate over target ids, and randomly pick source ids - since ideally we want the decoder-discriminator pairs trained with content variety